app/views/home/policy.html.erb

```diff
@@ -0,0 +1,134 @@
+         <h1 class="disclosures__one-line">Third-party disclosure</h1>
+           <p>We do not sell, trade, or otherwise transfer to outside parties your Personally Identifiable Information unless we provide users with advance notice. This does not include website hosting partners and other parties who assist us in operating our website, conducting our business, or serving our users, so long as those parties agree to keep this information confidential. We may also release information when it's release is appropriate to comply with the law, enforce our site policies, or protect ours or others' rights, property or safety.
+         <br>
+          <p>However, non-personally identifiable visitor information may be provided to other parties for marketing, advertising, or other uses.</p>
+       </div>
+
+       <div class="col-md-12 disclosures__information">
+         <h1 class="disclosures__one-line">Third-party links</h1>
+         <p>Occasionally, at our discretion, we may include or offer third-party products or services on our website. These third-party sites have separate and independent privacy policies. We therefore have no responsibility or liability for the content and activities of these linked sites. Nonetheless, we seek to protect the integrity of our site and welcome any feedback about these sites.</p>
+       </div>
+
+       <div class="col-md-12 disclosures__information">
+         <h1 class="disclosures__one-line">California Online Privacy Protection Act (CalOPPA)</h1>
+           <p>CalOPPA is the first state law in the nation to require commercial websites and online services to post a privacy policy. The law&apos;s reach stretches well beyond California to require any person or company in the United States (and conceivably the world) that operates websites collecting Personally Identifiable Information from California consumers to post a conspicuous privacy policy on its website stating exactly the information being collected and those individuals or companies with whom it is being shared. - See more at: <a href="http://consumercal.org/california-online-privacy-protection-act-caloppa/#sthash.0FdRbT51.dpuf">California Online Protection Website</a>
+           </p>
+         <h2 class="disclosures__title">According to CalOPPA, we agree to the following:</h2>
+           <p class="disclosures_paragraph-space">Users can visit our site anonymously.</p>
+           <p class="disclosures_paragraph-space">Once this privacy policy is created, we will add a link to it on our home page or as a minimum, on the first significant page after entering our website.</p>
+           <p class="disclosures_paragraph-space">Our Privacy Policy link includes the word &apos;Privacy&apos; and can easily be found on the page specified above.</p>
+           <p class="disclosures_paragraph-space">You will be notified of any Privacy Policy changes:</p>
+             <li> On our Privacy Policy Page </li>
```

Comment:
@emilyspieler can you review the indentation of the tags on this page? Indentation usually implies some sort of nesting, but there's a lot of indentation on this page that aren't really nested elements. For instance, the indentation of this `<li>` seems like it should be nested in the `<p>` tag above it, but it isn't. I don't think indentation is needed there. 
I think rubocop might be able to detect some of these things, but we don't have that yet.

Reply:
@markdamobi yeah rubocop will take care of this. After I merge this in I'll pull master and rerun rubocop before merging my PR to master.